1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:44:42.973448 I | embed: rejected connection from \"127.0.0.1:63216\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:44:42.973807       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:44:49.544773 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:44:54.327627       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:44:54.327784       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:44:55.046227       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"time=\"2021-12-01T11:44:58Z\" level=error msg=\"Port 30000 for service reactapp-service is already opened by another service\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:44:59.544330 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:02.801348       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:02.801501       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:02.802764       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/apiserver.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:02.804095       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/front-proxy-client.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:02.817947       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/front-proxy-ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:02.818024       1 dynamic_cafile_content.go:202] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:06.783479       1 warnings.go:70] discovery.k8s.io/v1beta1 EndpointSlice is deprecated in v1.21+, unavailable in v1.25+; use discovery.k8s.io/v1 EndpointSlice\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:06.950557       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:06.950592       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:06.950799       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:06.950843       1 dynamic_serving_content.go:165] key failed with : open /run/config/pki/ca.crt: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:45:09.544762 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:11.869958       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:11.874915       1 reflector.go:334] github.com/moby/vpnkit/go/cmd/kube-vpnkit-forwarder/main.go:49: watch of *v1.Service ended with: The resourceVersion for the provided watch is too old.\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"log: exiting because of error: log: cannot create log: open /tmp/kube-vpnkit-forwarder.vpnkit-controller.unknownuser.log.WARNING.20211201-114511.1: no such file or directory\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:15.226262       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:15.859953       1 horizontal.go:227] failed to query scale subresource for Deployment/default/dj-app-deployment: deployments/scale.apps \"dj-app-deployment\" not found\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:45:15.860930       1 event.go:291] \"Event occurred\" object=\"default/dj-app-hpa\" kind=\"HorizontalPodAutoscaler\" apiVersion=\"autoscaling/v2beta2\" type=\"Warning\" reason=\"FailedGetScale\" message=\"deployments/scale.apps \\\"dj-app-deployment\\\" not found\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"E1201 11:45:16.433146       1 horizontal.go:227] failed to query scale subresource for Deployment/default/fl-app-deployment: deployments/scale.apps \"fl-app-deployment\" not found\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:45:16.433408       1 event.go:291] \"Event occurred\" object=\"default/fl-app-hpa\" kind=\"HorizontalPodAutoscaler\" apiVersion=\"autoscaling/v2beta2\" type=\"Warning\" reason=\"FailedGetScale\" message=\"deployments/scale.apps \\\"fl-app-deployment\\\" not found\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:45:19.550215 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:45:25.905972       1 client.go:360] parsed scheme: \"passthrough\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:45:25.906123       1 passthrough.go:48] ccResolverWrapper: sending update to cc: {[{https://127.0.0.1:2379  <nil> 0 <nil>}] <nil> <nil>}\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"I1201 11:45:25.906150       1 clientconn.go:948] ClientConn switching balancer to \"pick_first\"\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:25.908796       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:45:25.907894 I | embed: rejected connection from \"127.0.0.1:63554\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:26.576590       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:45:26.910838 I | embed: rejected connection from \"127.0.0.1:63564\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:26.911690       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:27.686641       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:28.413617       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:45:28.413784 I | embed: rejected connection from \"127.0.0.1:63578\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:45:29.551797 I | etcdserver/api/etcdhttp: /health OK (status code 200)\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:30.854573       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:45:30.853884 I | embed: rejected connection from \"127.0.0.1:63598\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:31.324045       1 watcher.go:220] watch chan error: etcdserver: mvcc: required revision has been compacted\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"2021-12-01 11:45:35.705659 I | embed: rejected connection from \"127.0.0.1:63640\" (error \"open /run/config/pki/etcd/server.crt: no such file or directory\", ServerName \"\")\n","stream":"stderr"}
1970-01-01T00:33:41+00:00	docker.logs	{"log":"W1201 11:45:35.706662       1 clientconn.go:1223] grpc: addrConn.createTransport failed to connect to {https://127.0.0.1:2379  <nil> 0 <nil>}. Err :connection error: desc = \"transport: authentication handshake failed: remote error: tls: internal error\". Reconnecting...\n","stream":"stderr"}
